templates/vue/src/components/TapestryModal.vue

```diff
@@ -0,0 +1,131 @@
+<template>
```

Comment:
Should this file not go into the lightbox folder instead? Not sure - just checking.

Reply:
I left it here because it might be used outside of a lightbox component. I'm not sure, but I think we can leave it here for the time being.